src/components/dashboard/index.js

```diff
@@ -32,6 +32,8 @@ import ids from "./ids";
 import * as constants from "./constants";
 import { getDashboard } from "../../serviceFacades/dashboard";
 
+const makeID = (name) => buildID(ids.BASE, name);
```

Comment:
Minor, but I think you could refactor this to:
```suggestion
const makeID = (...args) => buildID(ids.BASE, ...args);
```

Reply:
This should be fixed now. Thanks for the suggestion.